### DISCUSSION TITLE
Kwil's next major release and it's impact on TSN

### DISCUSSION THREAD

**Discussion by zolotokrylin** [trufnetwork/node#138] [General]
@itscameronlee @brennanjl @rsoury please let's organise the call. Related to: 
- https://github.com/truflation/tsn-db/issues/114#issuecomment-2008592909

Please share your Calendly with me and I will book everyone based on the availability. Please make sure you consider that we are all in diff timezones, so keep your calendar availability options as wise as possible 🙏 

Cam, I already have your calendly.

**Comment by brennanjl** [comment]
https://savvycal.com/brennanbrennanBrennan-Lamey-ce54e68c/late

**Comment by rsoury** [comment]
https://calendly.com/ryansoury/truflation-x-usher-labs-engineering - for Raffael and I

https://calendly.com/ryansoury/chat - just for me

**Comment by brennanjl** [comment]
Hey @zolotokrylin are we still going to do this? Wanted to ensure this didn't get dropped.

**Comment by zolotokrylin** [comment]
@brennanjl @rsoury 
I have scheduled the meeting for 5th of April based on everyone's calendar availability.

I couldn't find time using your join calendar with Raffael, unfortunately.

**Reply by brennanjl** [reply]
I can try to make some other times, if you have suggestions? Just so that we can get something a bit sooner.

**Reply by brennanjl** [reply]
I'm in CST. I can do 8 AM-2 AM, if you give me enough of a heads up

**Reply by zolotokrylin** [reply]
Let's stick to that time please, because there are other unresolved issue we already need to take care of in TST and with Metric. 
🙏
